test/Libraries/Microsoft.Extensions.AI.Abstractions.Tests/ChatCompletion/StreamingChatCompletionUpdateTests.cs

```diff
@@ -91,8 +91,8 @@ public void Text_GetSet_UsesFirstTextContent()
             Role = ChatRole.User,
             Contents =
             [
-                new AudioContent("http://localhost/audio"),
-                new ImageContent("http://localhost/image"),
+                new DataContent("http://localhost/audio", mediaType: "audio/mpeg"),
+                new DataContent("http://localhost/image", mediaType: "image/png"),
```

Comment:
Did these tests actually require providing a media type now when wasn't provided before?

Reply:
No, I was just did this to replicate how we'd expect image and audio content to be created in the real world (it's more important now that `mediaType` is specified, since it's the main way to determine the content's type).

Happy to revert this change if it just adds confusion.

Also - should we consider removing the default value for the `mediaType` constructor parameter? This makes it so that you have to deliberately specify a `null` media type if you want one. It might reduce the chance that someone would leave it out by accident.

Reply:
Update: Removed `mediaType` where isn't not needed.